wave_eq_ncells_600_order_0_ssrk2/wave_eq_ncells_600_order_0_ssrk2.pvd

```diff
@@ -0,0 +1,206 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Remove simulation results from the repo. (I guess that you accidentally pushed them).

Reply:
Add vtu, pvd and jld in gitignore file so that we avoid this in the future.

Reply:
Done!